tools/MGnify_search.md

```diff
@@ -51,31 +121,26 @@ There different databases a user can choose to search their query against:
   
 In this example, we will search only for full length sequences.
 
-![database options under sequence subset](img/mgnify_search/sequence_subset.png)
-
-### 4. Result page
+#### 4. Result page
 
 A list of matching sequences is presented that is ordered by E-value significance (lower E-value indicating a more significant match). The user can customize this table to display different headings. Clicking on the `Target` link shows the amino acid FASTQ file that the query sequence matched to. the `Run & Sample IDs` link brings up the associated MGnify Sample Overview.
 
 In our [example](https://www.ebi.ac.uk/metagenomics/sequence-search/results/7CFA9DC8-900B-11EE-B07A-5352765DAC3F/score), 2368 significant query matches were found. The top match has a bit score of 890.8 and an E-value of 3.5e-265.
```

Comment:
Links from "fixed" results are removed after a certain time period. Clear dead links please and make them plaintext.